controllers/localregistry/clusterNativeLocalDoguRegistry.go

```diff
@@ -145,15 +146,16 @@ func (cmr *clusterNativeLocalDoguRegistry) getCurrentByDoguResource(ctx context.
 // GetCurrentOfAll retrieves the specs of all dogus' currently installed versions
 // through the ConfigMaps referenced in the specLocation field of the dogu resources' status.
 func (cmr *clusterNativeLocalDoguRegistry) GetCurrentOfAll(ctx context.Context) ([]*core.Dogu, error) {
-	doguList, err := cmr.doguClient.List(ctx, metav1.ListOptions{})
+	allLocalDoguRegistriesSelector := fmt.Sprintf("%s=%s,%s,%s=%s", appLabelKey, appLabelValueCes, doguNameLabelKey, typeLabelKey, typeLabelValueLocalDoguRegistry)
```

Comment:
Does the selector without the dogu.name value work? I guess this means that the field just exists without any value?